static/js/form-helpers.js

```diff
@@ -29,24 +30,27 @@ window.onload = (event) => {
             choiceSelect.value = 'other';  // Reset to default or empty value
             department_label.classList.remove('required');
         }
-
-        if (shouldHide) {
-            request_account_field.classList.add('hidden');
-        } else {
-            request_account_field.classList.remove('hidden');
-            request_account_label.classList.add('required');
+        if (request_account_field){
```

Comment:
Do I get it right, that this check is for this?
<img width="592" alt="image" src="https://github.com/user-attachments/assets/5ddf59d2-e0f0-49e8-8283-f9a76fd654a3">

If so, could you please leave a small comment in the source about it:)

Reply:
Thanks. Yes. It is because of this. I will add a comment.